src/Components/Blazor/Build/test/RenderingRazorIntegrationTest.cs

```diff
@@ -365,19 +365,19 @@ public async Task SupportsTwoWayBindingForTextboxes()
             var renderer = new TestRenderer();
 
             // Assert
-            Action<UIEventArgs> setter = null;
+            EventCallback setter = default;
             var frames = GetRenderTree(renderer, component);
             Assert.Collection(frames,
                 frame => AssertFrame.Element(frame, "input", 3, 0),
                 frame => AssertFrame.Attribute(frame, "value", "Initial value", 1),
                 frame =>
                 {
                     AssertFrame.Attribute(frame, "onchange", 2);
-                    setter = Assert.IsType<Action<UIEventArgs>>(frame.AttributeValue);
+                    setter = Assert.IsType<EventCallback>(frame.AttributeValue);
                 });
 
             // Trigger the change event to show it updates the property
-            await renderer.Invoke(() => setter(new UIChangeEventArgs { Value = "Modified value", }));
+            await renderer.InvokeAsync(() => setter.InvokeAsync(new UIChangeEventArgs { Value = "Modified value", }));
```

Comment:
Is the new `await` for convenience, or is the synchronous outcome actually different?

Assuming we haven't changed the event handling to now only take effect synchronously, it would be good to have at least one test for binding that is really explicit about what happens synchronously vs asynchronously (if there isn't already one).

Reply:
It's for thoroughness in tests. I'll update these to verify that it completed synchronously.